examples/AspNetCore/OData/ODataOpenApiExample/ConfigureSwaggerOptions.cs

```diff
@@ -50,25 +50,84 @@ private static OpenApiInfo CreateInfoForApiVersion( ApiVersionDescription descri
             text.Append( " This API version has been deprecated." );
         }
 
-        if ( description.SunsetPolicy is { } policy )
+        if ( description.DeprecationPolicy is { } deprecationPolicy )
         {
-            if ( policy.Date is { } when )
+            if ( deprecationPolicy.Date is { } when )
```

Comment:
Shouldn't these messages expand rather than be replaced? They are complementary with different meanings.

- **Deprecated** - obsolete and not recommended
- **Sunset** - permanently removed

An API _could_ be deprecated forever, but if it does have a sunset policy, then that indicates when it will go away - forever.